File: program-manager_sheduler/promanager-ui/ui/src/app/model/session.ts
export interface  Session {
  id: string;
  programId: string;
  startDate: Date;
  endDate: Date;
  text: string;
}

// tslint:disable-next-line: class-name
export interface sessionData {
  text: string;
  startdate: Date;
  enddate: Date;
}

File: program-manager_sheduler/api/src/main/java/in/stackroute/planage/promanager/model/Session.java
package in.stackroute.planage.promanager.model;

import lombok.*;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Document
public class Session {
@Id
  private String id = UUID.randomUUID().toString();
  private String programId;
  private String text;
  private Date startDate;
  private Date endDate;

}
